examples/ultrahdr_app.cpp

```diff
@@ -1172,37 +1172,44 @@ void UltraHdrAppInput::computeYUVSdrPSNR() {
 
 static void usage(const char* name) {
   fprintf(stderr, "\n## ultra hdr demo application.\nUsage : %s \n", name);
-  fprintf(stderr, "    -m    mode of operation. [0:encode, 1:decode] \n");
+  fprintf(stderr, "    -m    mode of operation, optional. [0:encode, 1:decode (default)] \n");
   fprintf(stderr, "\n## encoder options : \n");
   fprintf(stderr, "    -p    raw 10 bit input resource \n");
   fprintf(stderr,
           "    -y    raw 8 bit input resource, optional. \n"
           "          if not provided tonemapping happens internally. \n");
   fprintf(stderr,
-          "    -a    raw 10 bit input resource color format. Required if raw hdr intent is "
-          "provided. [0:p010, 5:rgb1010102] \n");
+          "    -a    raw 10 bit input resource color format, optional. [0:p010, 5:rgb1010102 "
+          "(default)] \n");
```

Comment:
Could you double check / confirm the default color format is RGB instead of YUV? And besides this should be `rgba1010102`, seems like letter 'a' is missing

Reply:
Dichen, I have addressed rgb1010102 to rgba1010102. I have checked and the default color format is rgb.